xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/hierarchy.vm

```diff
@@ -32,6 +32,9 @@
     ##
     ## Default mode: display the hierarchy for the current page
     ##
-    #hierarchy($NULL {'id': 'hierarchy', 'limit': 5, 'treeNavigation': true})
+    #hierarchy($NULL {'id': 'hierarchy',
+    'navigation': true,
+    'label': $doc.getPageReference().getName(),
```

Comment:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/2763c7af3c83cd0ca2297a779db4d435168ef8ff :+1: 
Since this is the only breadcrumb with the navigation role left, there is no need to find a more precise name than 'Breadcrumb' :)